dashboard/db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
there should be no changes to this (autogenerated) file other than syntax/formatting changes.